packages/widgetbook/lib/src/addons/addons.dart

```diff
@@ -1,40 +1,63 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 import 'package:widgetbook/widgetbook.dart';
 
 export 'package:widgetbook_addon/src/widgetbook_addon.dart';
+
 export './cupertino_theme_addon/addon.dart';
 export './cupertino_theme_addon/cupertino_theme_addon.dart';
 export './custom_theme_addon/addon.dart';
+export './device_addon/addon.dart';
 export './localization_addon/addon.dart';
 export './material_theme_addon/addon.dart';
 export './material_theme_addon/material_theme_addon.dart';
 export './text_scale_addon/addon.dart';
 export './theme_addon/addon.dart';
-export './device_addon/addon.dart';
 
 List<WidgetbookAddOn> configureMaterialAddons({
-  required MaterialThemeSetting themeSetting,
-  required TextScaleSetting textScaleSetting,
-  required LocalizationSetting localizationSetting,
+  required List<WidgetbookTheme<ThemeData>> themes,
+  required List<double> textScales,
+  required List<Locale> locales,
+  required List<LocalizationsDelegate> localizationsDelegates,
   required List<Device> devices,
 }) {
   return [
-    MaterialThemeAddon(setting: themeSetting),
-    TextScaleAddon(setting: textScaleSetting),
-    LocalizationAddon(setting: localizationSetting),
-    DeviceAddon(devices: devices),
+    MaterialThemeAddon(
```

Comment:
I'm not sure if we need this method at all. 
In addition, this does not allow to specify what's selected as the default/on startup of Widgetbook

Reply:
This method will come in handy in the future, when the use is the one responsible for adding the addons themselves (without code generation), then they can add all default addons with it.

Reply:
I think this method does not work well if you also want an initial configuration